yt_dlp/extractor/platzi.py

```diff
@@ -111,17 +111,16 @@ def _real_extract(self, url):
         duration = video_player.get('duration', '')
         servers = try_get(video_player, lambda x: x['video']['servers'])
         formats = []
-        for server in servers.keys():
-            server_json = servers.get(server, {})
-            if 'hls' in server_json.keys():
+        for _, server_json in servers.items():
```

Comment:
oh, I didnt realize `server` was unused. In that case, better to do 
```suggestion
        for server_json in servers.values():
```